src/teleop/jetson/jetson_teleop.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+#Node for teleop-related callback functions
+
+from math import copysign
+from enum import Enum
+import rospy as ros
+from sensor_msgs.msg import Joy
+from geometry_msgs.msg import Twist
+from mrover.msg import DriveVelCmd, RAOpenLoopCmd, HandCmd
+
+
+def quadratic(val):
+    return copysign(val**2, val)
+
+def deadzone(magnitude, threshold):
+    temp_mag = abs(magnitude)
+    if temp_mag <= threshold:
+        temp_mag = 0
+    else:
+        temp_mag = (temp_mag - threshold)/(1 - threshold)
+
+    return copysign(temp_mag, magnitude)
+
+
+class Drive:
+    class LogitechAxes(Enum):
+      left_right = 0
+      forward_back = 1
+      twist = 2
+      dampen = 3
+      pan = 4
+      tilt = 5
+
+    #TODO: Reimplement Dampen Switch
+    def teleop_drive_callback(self, msg):
+
+        #teleop_twist_joy message is maxed at 0.5, multiply by 2 to make range [-1,1]
```

Comment:
is it asserted that teleop_twist_joy is maxed at 0.5? the comment is very helpful but it's kind of interesting that it's like this

Reply:
Turns out that the teleop_twist_joy node is set up for both a "turbo" mode that maxes at 1 and a "regular speed" mode that maxes at 0.5 depending on the buttons you set up in the node params. I will just make the fake button that gets pushed to the front of the button array enable turbo instead and I believe it will get rid of the need to multiply by 2.

Reply:
interesting lol. sounds good.

Reply:
for clarifcation: so this means by default we will be using turbo mode? and does the operator have a choice?

Reply:
Yeah this means by default teleop_twist_joy is in "turbo" mode, it could be changed so that the operator has an option either using a joystick button or a button on the GUI. Also, I came to realize that this turbo mode only affects the linear component of the Twist velocity, so in the new commits the angular velocity is still multiplied by 2 to maintain the [-1,1] range.

Reply:
sounds good